mpas_analysis/sea_ice/climatology_map_risk_index_outcome.py

```diff
@@ -0,0 +1,300 @@
+            A list of variable names in ``timeSeriesStatsMonthly`` to be
+            included in the climatologies
+        seasons : list of str, optional
+            A list of seasons (keys in ``shared.constants.monthDictionary``)
+            to be computed or ['none'] (not ``None``) if only monthly
+            climatologies are needed.
+        comparison_grid_names : list of {'latlon', 'antarctic'}
+            The name(s) of the comparison grid to use for remapping.
+        """
+
+        subtask_name = f'remapMpasClimatology_RiskIndexOutcome'
+        # call the constructor from the base class
+        # (RemapMpasClimatologySubtask)
+        super().__init__(
+            mpas_climatology_task, parent_task, climatology_name,
+            variable_list, seasons, comparison_grid_names)
+
+        self.mpas_climatology_task = mpas_climatology_task
+        self.variable_list = variable_list
+
+    def setup_and_check(self):
+        """
+        Perform steps to set up the analysis and check for errors in the setup.
+        """
+
+        # first, call setup_and_check from the base class
+        # (RemapMpasClimatologySubtask), which will set up remappers and add
+        # variables to mpas_climatology_task
+        super().setup_and_check()
+
+        # don't add the variables and seasons to mpas_climatology_task until
+        # we're sure this subtask is supposed to run
+        self.mpas_climatology_task.add_variables(self.variable_list,
+                                                     self.seasons)
+
+    def customize_masked_climatology(self, climatology, season):
+        """
+        Compute the risk index outcome from sea-ice concentration and (floe) thickness.
+
+        Parameters
+        ----------
+        climatology : xarray.Dataset
+            the climatology data set
+        season : str
+            The name of the season to be masked
+        Returns
+        -------
+        climatology : xarray.Dataset
+            the modified climatology data set
+        """
+
+        rio = self._compute_risk_index_outcome(climatology)
+
+        climatology['riskIndexOutcome'] = rio
+        climatology.riskIndexOutcome.attrs['units'] = ''
+        climatology = climatology.drop_vars(self.variable_list)
+
+        return climatology
+
+    def _compute_risk_index_outcome(self, climatology):
+        """
+        Compute the risk index outcome from sea-ice concentration and (floe) thickness,
+        as outlined in the International Maritime Organization (IMO) document.
+        (https://www.imorules.com/GUID-2C1D86CB-5D58-490F-B4D4-46C057E1D102.html)
+        """
+        ds_restart = xr.open_dataset(self.restartFileName)
+        ds_restart = ds_restart.isel(Time=0)
+
+        # sea-ice concentration conversion from range [0,1] to range [0,10]
+        scale_factor = 10
+        # polar class array index should be in the range [0,11], but not checked!
+        pc = self.config.getint(self.taskName, 'polarClass') - 1
```

Comment:
It there a reason not to produce plots for all 12 classes?  If yes, could there be a desire to produce plots for more than one class, in which case `polarClass` should become a list of `polarClasses`?